busy-app/busybar-firmware
# Ble PER test app

Add app BLE PER test

**Testing**
1. send U5 to permanent reset
2. activate CLI on 917
3. connect TTL converter to U5 debug port (tx-tx, rx-rx) speed 230400
4. run CLI application on 917 "ble_per_test"
5. run SDR on 2412 MHz with max bandwidth
6. check application operability (commands are described in console). change of channels and signal mode should be correctly displayed in SDR.
7. to conduct a batch test it is necessary to run two 917, on one select "tx", on 2 "rx". channels, signal strength, transmission patterns as desired.
